apps/daimo-mobile/package.json

```diff
@@ -9,7 +9,7 @@
     "build:prod": "eas build -p all --profile=production --auto-submit --non-interactive",
     "submit:prod": "eas submit --platform ios --profile=production",
     "submit:prod-android": "eas submit --platform android --profile=production",
-    "dev": "DAIMO_APP_API_URL=http://`ipconfig getifaddr en0`:3000 expo start --dev-client --clear -i",
+    "dev": "DAIMO_APP_API_URL=`ipconfig getifaddr en0`:3000 expo start --dev-client --clear -i",
```

Comment:
does this work for you?

for me, i see following:
```
$ ipconfig getifaddr en0
192.168.1.22
```

Reply:
my bad, did not correctly revert the commit where I made changes for testing.